doc/reference/configuration/configuration_reference.rst

```diff
@@ -2380,6 +2380,107 @@ The ``memtx`` section is used to configure parameters related to the :ref:`memtx
     | Default: box.NULL
     | Environment variable: TT_MEMTX_SORT_THREADS
 
+..  _configuration_reference_process:
+
+process
+-------
+
+The ``process`` defines configuration parameters of the Tarantool process in the system.
+
+
+-   :ref:`process.background <configuration_reference_process_background>`
+-   :ref:`process.coredump <configuration_reference_process_coredump>`
+-   :ref:`process.custom_proc_title <configuration_reference_process_custom_proc_title>`
```

Comment:
Looks like it should be `title`:

```suggestion
-   :ref:`process.title <configuration_reference_process_title>`
```

Here is the `tarantool --help-env-list` output:

```
tarantool --help-env-list | grep PROCESS
| TT_PROCESS_BACKGROUND                                 | boolean        | false                                         | Community Edition  |
| TT_PROCESS_COREDUMP                                   | boolean        | false                                         | Community Edition  |
| TT_PROCESS_PID_FILE                                   | string         | var/run/{{ instance_name }}/tarantool.pid     | Community Edition  |
| TT_PROCESS_STRIP_CORE                                 | boolean        | true                                          | Community Edition  |
| TT_PROCESS_TITLE                                      | string         | tarantool - {{ instance_name }}               | Community Edition  |
| TT_PROCESS_USERNAME                                   | string         | box.NULL                                      | Community Edition  |
| TT_PROCESS_WORK_DIR                                   | string         | box.NULL                                      | Community Edition  |
```

Reply:
Thanks. I've copied the wrong field name from the source code.